students/gyeongjae-ham/westagram/users/views.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from django.http     import JsonResponse
+from django.views    import View
+
+from .models import User
+
+class CreateUserView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if User.objects.filter(email=data['email']).exists() == True:
```

Comment:
User.objects.filter(email=data['email']).exists()의 return 값이 True 또는 False 입니다. 
따라서 == True 없이도 조건문을 판단 할 수 있습니다.
```suggestion
            if User.objects.filter(email=data['email']).exists():
```